### PR TITLE
Add RFC 2119 recommendation to EIP-1

### LINKED ISSUE #2960: EIP-1: Include RFC 2119 recommendation into EIP-1 so EIPs do not need to duplicate it
Pretty much every "seriously written" EIP duplicates that paragraph from RFC 2119.

### PR DESCRIPTION
**Pull request by axic**
Fixes #2960

### CODE REVIEW

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -247,8 +247,16 @@ The editors don't pass judgment on EIPs. We merely do the administrative & edito
 
 ## Style Guide
 
+### EIP numbers
+
 When referring to an EIP by number, it should be written in the hyphenated form `EIP-X` where `X` is the EIP's assigned number.
 
+### RFC 2119
+
+EIPs are suggested to follow [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for terminology and to insert the following at the beginning of the Specification section:
```
Comment (axic) [inline EIPS/eip-1.md:256]:
```suggestion
EIPs are encouraged to follow [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for terminology and to insert the following at the beginning of the Specification section:
```

**Comment by lightclient** [comment]
LGTM

**Review by MicahZoltu — APPROVED** [review]
You decided not to add it to the template?

**Comment by axic** [comment]
> You decided not to add it to the template?

I can add it if it makes sense.

**Comment by MicahZoltu** [comment]
I think it makes sense to put it in the template, at the start of the Specification section (above the comment).  Should I wait to merge this or do you want to do that as a separate PR?

**Comment by axic** [comment]
Added, authors can choose to remove it anyway.

**Review by MicahZoltu — APPROVED** [review]

**Pull request merged**